src/useCdn.ts

```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react'
+
+export default function useCdn(): [string, (cdn: string) => void] {
```

Comment:
Just wondering. There isnt really much logic to this hook, i kinda think we can just perform the update in the file itself?

Reply:
The update isn't made on the file; `scripts.js` but on the `useLazerpay` file.